src/style.css

```
@@ -1,3 +1,72 @@
+:root {
+  --link-color: #00ACDC;
+  --text-color: #000;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.list {
+  display: flex;
+  align-items: center;
+  list-style-type: none;
+  text-transform: uppercase;
+  margin: 0;
+  padding: 0;
+}
+
+.elements {
+  margin: 0 10px;
+  position: relative;
+}
+
+.link{
+  text-decoration: none;
+  display: flex;
+  text-align: center;
+  line-height:60px;
+  height: 60px;
+  color: var(--text-color);
+}
+
+.elements:first-child {
+  margin-left: 0;
+}
+
+.elements:last-child {
+  margin-right: 0;
+}
+
+.link:hover {
+  color: var(--link-color);
+}
+
+.link:hover,
+.link.is-active {
+  color: var(--link-color);
+}
+
+.link.is-active::after {
```

you can do it just with `.is-active` class?
```suggestion
.is-active::after {
```